runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -1560,6 +1560,7 @@ bool handleServerMessage(JITaaS::J9ClientStream *client, TR_J9VM *fe)
          break;
       case J9ServerMessageType::ResolvedMethod_isUnresolvedString:
          {
+         TR_ASSERT(false, "ResolvedMethod_isUnresolvedString is combined into ResolvedMethod_stringConstant");
```

Comment:
Can we simply remove the message or are you concerned that we can still reach this line and the corresponding one in `TR_ResolvedJ9JITaaSServerMethod::isUnresolvedString()`, hence the asserts? (If so, an issue tagged with beginner to clean it up later might be a good way to take care of it. :grin:)

Reply:
At the moment, `isUnresolvedString()` is always called right after `stringConstant()` in `J9::SymbolReferenceTable::findOrCreateStringSymbol()`. The concern is that when these two methods are decoupled in the non-jitaas code in the future for any reason, and if the `ResolvedMethod_isUnresolvedString` message is removed, it will break the code. It'll be hard to track the bug down. I will open a beginner issue to evaluate if this message should be removed or at least the related assert should be removed.